File: ProgLangChallenge/Java/Driver.java
import java.util.*; 

public class Driver
{
    public static void main(String[] args)  
    {
        CSVReader reader = new CSVReader();
        reader.readFile("/home/deep/ProgLang/ProgLangChallenge/Input/input2.csv");
        
        VoterCandidate vc = new VoterCandidate();
        List<List<String>> nameScores  = vc.getNamesAndScores(reader.getVoter(), reader.getCandidates());
        int highestScore = vc.findHighestScore(nameScores);
        List<String> bestCandidates = vc.getCandidatesWithHishestScore(nameScores, highestScore);
        System.out.println(bestCandidates);
    }
}

File: ProgLangChallenge/C++/Driver.cpp
#include <iostream>
#include "CSVReader.h"
#include "VoterCandidate.h" 
using namespace std;

int main()
{
    CSVReader reader; 
    reader.readFile("../Input/input3.csv");

    VoterCandidate vc; 
    vector<vector<string>> nameScores = vc.getNamesAndScores(reader.getVoter(), reader.getCandidates());
    int highestScore = vc.findHighestScore(nameScores);
    vector<string> bestCandidates = vc.getCandidatesWithHishestScore(nameScores, highestScore);
    reader.print1DVector(bestCandidates);

    return 0;
}
